README.md

```diff
@@ -1,208 +1,262 @@
-# Mendix Push notifications
+# Mendix Push Notifications
 
-This module and widget should be used to implement push notifications for android, ios and windows tablets/mobiles. This module/widget has been tested to work on Mendix 5.14.1 and up. 
+Push notifications let your application notify a user of events even when the user is not actively using the application. This is a native capability provided by both Android and iOS devices and made available via Google Cloud Messaging (GCM) and Apple Push Notifications service (APNs). This project is meant to make it easy for Mendix developers who want to include Push Notifications capability into their Mendix hybrid mobile application.
 
-The project contains:
+## Overview
 
-- Directory structure.
-- Readme.md file.
-- Javascript source.
-- XSD for package.xml, to configure properties of the widget, visible inside the Mendix business modeler.
-- Example project
-- Module for sending push notifications
-- Custom css file to style the push notification html
+<img src="assets/images/overview/architecture.png"/>
```

Comment:
Could update with the colored picture I created for the presentation, might be a bit easier to read